lib/stacker/stack/template.rb

```diff
@@ -42,7 +56,11 @@ def remote
       end
 
       def diff *args
-        Differ.json_diff local, remote, *args
+        if json?
+          Differ.json_diff local, remote, *args
```

Comment:
The normalization before diffing is nice, but is it necessary? I'd prefer to just diff the raw templates for both JSON and YAML.